src/modules/getcomment.js

```diff
@@ -0,0 +1,34 @@
+import countComment from './commentCount.js';
+
+const getComment = async () => {
+  const commentId1 = document.querySelector('.idMeal');
+  const commentIdval = commentId1.innerHTML;
+  fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/UeRVpttbuH5idOpNpTZz/comments?item_id=${commentIdval}`, { mode: 'cors' }, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  })
```

Comment:
- [Required] I think since this function fetches some data from an `API` online and it might take some time it would be better if you use  `await` with the `fetch` method to receive the data from the API. The keyword `async` before a function makes the function return a promise and the `await` keyword makes the function pause the execution and wait for a resolved promise before it continues which in turn makes your function run synchronously without an error. Please see my first comment in the review on how you can make this function use `async` and `await`.